fix(core): :bug: notificationpopover is design is completed properly

arkaprota opened the pull request:
## Description

- [ ] NotificationPopOver is not designed properly
- [ ] the icon is changed
- [ ] the heading alignment is not given properly

Ref: #373


## Screenshots (if applicable)

![image](https://github.com/user-attachments/assets/94c9ba57-d1a6-48a8-8f5c-b5a4f9f70135)

arkaprota committed:
fix(core): :bug: notificationpopover is design is completed properly

notificationpopover has been fixed

Ref: #373

techoneel commented on package/components/utils/NotificationPopOver.js:
```diff
@@ -13,18 +13,19 @@ export default function NotificationPopOver() {
 
           <CoreIconButton disabled>
-            <CoreIcon>clear_all</CoreIcon>
+            <CoreIcon>more_horiz</CoreIcon>
```
who ask you to change this

arkaprota replied:
i saw that it was looking bad thats why i change it if you want i will revert it back to the previous one

arkaprota replied:
i have resolve the issue please see through this

techoneel committed:
Merge branch 'development' into WRPD-bugfix-373

arkaprota committed:
fix(global): :bug: change the coreicon more_horiz to clear_all which was previouly given

the coreicon more_horiz to clear_all which was there previouly was been required is solved

Ref: #373

arkaprota committed:
Merge branch 'development' of https://github.com/wrappid/core into WRPD-bugfix-373

arkaprota committed:
Merge branch 'WRPD-bugfix-373' of https://github.com/wrappid/core into WRPD-bugfix-373

arkaprota committed:
Merge branch 'development' into WRPD-bugfix-373

techoneel commented on package/components/utils/NotificationPopOver.js:
```diff
@@ -13,18 +13,19 @@ export default function NotificationPopOver() {
+
       <CoreBox styleClasses={[CoreClasses.POPOVER.CONTENT]}>
         <CoreTypographyBody1
           sx={{ minWidth: "350px" }}
```
cannot support sx property

techoneel replied:
try to avoid padding and margin to center aligned a content or make it wider or taller

arkaprota replied:
i have changed the sx property

techoneel reviewed (changes requested):
Please do the needful

arkaprota committed:
Merge branch 'development' into WRPD-bugfix-373

arkaprota committed:
fix(core): :bug: remove the sx

remove the sx property

Ref: #373
